README.md

```diff
@@ -1,37 +1,21 @@
 # PlaceCal
 
-## Introduction / Warning
+## Introduction
 
-PlaceCal is a large and very complicated app for collating organisation and event information from a large variety of sources. In the words of one developer: "I'm not sure where the interface between the app and the real world is here".
+PlaceCal is an online calendar which lists events and activities by and for members of local communities, curated around interests and locality.
+
+The codebase doesn't currently have enough seeds to create a dev environment suitable for developing from scratch. If you're interested in contributing to PlaceCal please get in touch with us at support@Placecal.org
 
 To get an idea of the project and what we're about, check out [the handbook](https://handbook.placecal.org/).
 
 ## Requirements
 
-To run PlaceCal locally you will need:
-
-- A Mac or a Linux machine (we don't support Windows at present)
-- GNU Compiler Collection (gcc) (for compiling ruby and gems)
-- Docker (optional) (for isolating and automating postgres management)
-- Postgres relational database. We are currently using v14.
-  - Server
-    - either installed for your distribution or as a docker image (with the correct open port -- see below)
-  - Client
-    - you will still need the local developer libraries for postgres
-    - these are distribution specific so you need to find out what they are called to install them
-      - `libpq-dev` (debian)
-      - `postgresql-libs` (arch)
-      - `dev-db/postgresql` (gentoo)
-- Ruby 3.1.2 - We recommend using a version manager for this such as `rvm` or `rbenv`. Current version we are using is in `.ruby-version`.
-  - [rvm](https://rvm.io/)
-  - [rbenv](https://github.com/rbenv/rbenv)
-    - [ruby-build](https://github.com/rbenv/ruby-build)
-    - [rbenv-gemset](https://github.com/jf/rbenv-gemset) (optional)
-- Node.js 16.x. We recommend using a version manager for this such as `nvm` or `nodenv`. Current version we are using is in `.node-version`.
-  - [nvm](https://github.com/nvm-sh/nvm)
-  - [nodenv](https://github.com/nodenv/nodenv)
+To run PlaceCal locally you will need to install the following dependencies:
+
+- Docker
```

Comment:
```suggestion
- [Docker](https://docs.docker.com/get-docker/)
```

Reply:
Oh I see you removed the links deliberately? Fine either way.

Reply:
I'll re-list them all as links, doesn't take up any more space or reading and if it might be useful to someone then it makes sense to include